mw-webapp/src/component/input/Input.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {HTMLInputTypeAttribute} from "react";
+import clsx from "clsx";
+import styles from "src/component/input/Input.module.scss";
+
+/**
+ * Defines what kind of input mode browser should present to the user
+ */
+export enum InputMode {
+  none = "none",
+  text = "text",
+  tel = "tel",
+  url = "url",
+  email = "email",
+  numeric = "numeric",
+  decimal = "decimal",
+  search = "search",
+}
+
+/**
+ * Input's props
+ */
+interface InputProps {
+  /**
+   * Input's value
+   */
+  value: string;
+  /**
+   * Input's type (what type of value is expected)
+   */
+  type?: HTMLInputTypeAttribute;
+
+  /**
+   * Input's placeholder text
+   */
+  placeholder?: string;
+
+  /**
+   * Input's class
+   */
+  className?: string | Array<string>;
+
+  /**
+   * Input's mode (defines what kind of input mode browser should present to the user)
+   */
+  inputMode?: InputMode;
+
+  /**
+   * The input usable or not
+   */
+  disabled?: boolean;
+
+  /**
+   * The input is required or not
```

Comment:
What does this mean, could you south not time thinking about comment. Use comment to simplify coffee no complexity. Check everything